multipers/mma_structures.pyx.tp

```diff
@@ -98,11 +99,22 @@ cdef inline get_summand_filtration_values_{{SHORT}}(Summand[{{CTYPE}}] summand):
     """
     Returns a list (over parameter) of the filtrations values of this parameter.
     """
-    cdef vector[Finitely_critical_multi_filtration[{{CTYPE}}]] v = summand.get_birth_list()
+    cdef vector[One_critical_filtration[{{CTYPE}}]] v = summand.get_birth_list()
     births = _vff21cview_{{SHORT}}(v, copy=True)
     v = summand.get_death_list()
     deaths = _vff21cview_{{SHORT}}(v, copy=True)
-    pts = np.concatenate([births,deaths],axis=0)
+
+    if len(births) == 1 and births[0].size == 1 and isinf(births[0][0]):
+        if len(deaths) == 1 and deaths[0].size == 1 and isinf(deaths[0][0]):
+            return []
```

Comment:
These checks should be done in c++ (we have access to `v`). Each `One_critical_filtration` has the is inf method

Reply:
So, should I add `is_plus/minus_inf()` to the Cython interface of `One_critical_filtration`? Is not there for now.